D:\src\hellner\trunk>ninja -C out/Debug
ninja: Entering directory `out/Debug'
[22/2280] RULE Generating C++ and Pyth...m audio_processing\test\unittest.proto
FAILED: C:\depot_tools\python276_bin\python.exe gyp-win-tool action-wrapper environment.x86 audioproc_unittest_proto_target_genproto_d946c28e9a060a0be378c58217d7941f.ee1175526f7cc19132e7e1558ed635c5.rsp ..\..\webrtc\modules
Traceback (most recent call last):
  File "../../tools/protoc_wrapper/protoc_wrapper.py", line 110, in <module>
    sys.exit(main(sys.argv))
  File "../../tools/protoc_wrapper/protoc_wrapper.py", line 95, in main
    ret = subprocess.call(protoc_args)
  File "C:\depot_tools\python276_bin\lib\subprocess.py", line 522, in call
    return Popen(*popenargs, **kwargs).wait()
  File "C:\depot_tools\python276_bin\lib\subprocess.py", line 709, in __init__
    errread, errwrite)
  File "C:\depot_tools\python276_bin\lib\subprocess.py", line 957, in _execute_c
hild
    startupinfo)
WindowsError: [Error 5] Access is denied
[22/2280] RULE Generating C++ and Pyth...code from audio_processing\debug.proto
FAILED: C:\depot_tools\python276_bin\python.exe gyp-win-tool action-wrapper environment.x86 audioproc_debug_proto_target_genproto_71eb4b4ece9debb5da06438d40b8da
f5.894d936f671a2292b086206746b9aa74.rsp ..\..\webrtc\modules
Traceback (most recent call last):
  File "../../tools/protoc_wrapper/protoc_wrapper.py", line 110, in <module>
    sys.exit(main(sys.argv))
